home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/PollQuestionServiceImpl.java

```diff
@@ -0,0 +1,159 @@
+package com.softserveinc.ita.homeproject.homeservice.service.impl;
+
+import com.softserveinc.ita.homeproject.homedata.entity.AdviceChoiceQuestion;
+import com.softserveinc.ita.homeproject.homedata.entity.MultipleChoiceQuestion;
+import com.softserveinc.ita.homeproject.homedata.entity.Poll;
+import com.softserveinc.ita.homeproject.homedata.entity.PollQuestion;
+import com.softserveinc.ita.homeproject.homedata.entity.PollQuestionType;
+import com.softserveinc.ita.homeproject.homedata.entity.PollStatus;
+import com.softserveinc.ita.homeproject.homedata.repository.PollQuestionRepository;
+import com.softserveinc.ita.homeproject.homedata.repository.PollRepository;
+import com.softserveinc.ita.homeproject.homeservice.dto.PollQuestionDto;
+import com.softserveinc.ita.homeproject.homeservice.dto.PollQuestionTypeDto;
+import com.softserveinc.ita.homeproject.homeservice.exception.NotFoundHomeException;
+import com.softserveinc.ita.homeproject.homeservice.exception.TypeOfTheContactDoesntMatchHomeException;
+import com.softserveinc.ita.homeproject.homeservice.mapper.ServiceMapper;
+import com.softserveinc.ita.homeproject.homeservice.service.PollQuestionService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+
+@Service
+@RequiredArgsConstructor
+public class PollQuestionServiceImpl implements PollQuestionService {
+
+    private final PollQuestionRepository pollQuestionRepository;
+
+    private final PollRepository pollRepository;
+
+    private final ServiceMapper mapper;
+
+    private static final String POLL_WITH_ID_NOT_FOUND = "Poll with 'id: %d' is not found";
```

Comment:
I suggest you use the constant NOT_FOUND_MESSAGE from the interface QueryableService, and build th string as String.format(NOT_FOUND_MESSAGE, "Poll", id). Haw you think? 
Next constant the same.

Reply:
Done